src/tcgwars/logic/impl/gen4/MysteriousTreasures.groovy

```diff
@@ -1234,8 +1234,8 @@ public enum MysteriousTreasures implements LogicCardInfo {
               checkLastTurn()
               checkNoSPC()
               powerUsed()
-              def chosenPCS = opp.all.select("Choose which Pokémon should have its types copied by Ninetales' $thisAbility.name?")
-              bc "$thisAbility.name - $chosenPCS was chosen"
+              def chosenPCS = opp.all.select("Choose which Pokémon should have its types copied by Ninetales' $thisAbility?")
```

Comment:
I'd probably change Ninetales to a `$self` as well.

Reply:
Good call 👍